tests/test_utils.py

```diff
@@ -89,8 +89,8 @@ def test_build_info_msg():
         "\n - 500 words, last updated on 2022-12-13 00:00:00"
         "\n"
         "\n Project hero contributors who deserve a cape:  "
-        f"\n {utils.hero_char} romanito"
-        f"\n {utils.hero_char} menganito"
+        f"\n {utils.hero_char} <a href=\"https://github.com/romanito\">romanito</a>"
```

Comment:
instead of escaping the double quotes you can make the f-string with single quotes so you avoid escaping. Which it makes it more readable

Reply:
same below